modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -220,6 +220,23 @@ The error codes include details of what error was encountered.
 [[online-backup-configurations]]
 == Online backup configurations
 
+[[online-backup-checkpoints]]
+=== Checkpoint configurations
```

Comment:
Maybe just Checkpointing, as we are not talking about how to configure backup checkpoints per se.
```suggestion
=== Checkpointing
```